frontend/src/components/pages/CreateReview/CreateReview.tsx

```diff
@@ -55,15 +53,15 @@ const CreateReview = ({ id }: CreateReviewProps): React.ReactElement => {
     onOpen: onOpenBookModal,
     onClose: onBookModalClose,
   } = useDisclosure();
+
   const [currBook, setCurrBook] = useState<Book | null>(null);
+  const newToast = ToastHook();
 
-  const [showDeleteBookModal, setShowDeleteBookModal] = useState<boolean>(
```

Comment:
Any clue why these lines were changed? Maybe try running the linter

Reply:
This one is giving me a headache, my vscode keeps changing it on save due to eslint/prettier and I agree with it in terms of style. Not sure why eslint in the docker does not like it.

Reply:
is your vscode using a different prettier config? you might have to add a project specific setting to use the prettier config from this project